perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -90,6 +94,27 @@ def AptInstall(vm):
   _Install(vm)
 
 
+def JujuInstall(vm):
+  if vm.isController is True:
+    if vm.vm_group_specs and 'workers' in vm.vm_group_specs:
+      vm.JujuDeploy('cs:trusty/cassandra',
+                    vm.vm_group_specs['workers'].vm_count,
+                    'workers')
```

Comment:
Here's the change I was describing: https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/938
Does that make sense now?